docs/primitives/index.md

```diff
@@ -0,0 +1,57 @@
+Primitives are basic 3d objects that can build more complex shapes.
+For example, a 3d plot can be converted into [polygons](polygon).
+
+If you are using this library as a backend to a Wolfram-Language-like project
+that supports [Plot3D](https://reference.wolfram.com/language/ref/Plot3D.html)
+you may convert that to a
+[Graphics3D](https://reference.wolfram.com/language/ref/Graphics3D.html)
+object (this library goal is to fully support it), or create a conversion
+function for every plot function.
+
+An *example* of conversion code for a
```

Comment:
I don't understand the italic here.

Reply:
In the current version of mathics-threejs-backend you need to invert the y and z coordinates to the plot to be "horizontal". But I plan to fix that next year.